tests/lowlevel_test.go

```diff
@@ -15,6 +14,10 @@ import (
 //
 // See https://github.com/gopherjs/gopherjs/issues/279.
 func TestTimeInternalizationExternalization(t *testing.T) {
+	if runtime.GOARCH == "js" {
```

Comment:
Same question on why the build tag isn't sufficient in this situation and why you prefer the `t.Skip`